overlord/fdestate/fdemgr.go

```diff
@@ -87,6 +88,28 @@ func Manager(st *state.State, runner *state.TaskRunner) (*FDEManager, error) {
 	defer st.Unlock()
 	st.Cache(fdeMgrKey{}, m)
 
+	snapstate.RegisterAffectedSnapsByKind("efi-secureboot-db-update", dbxUpdateAffectedSnaps)
+
+	// TODO call to state.RegisterPendingChangeByAttr() to block state from
+	// pruning EFI changes?
```

Comment:
not needed with cleanup and & undo now?

Reply:
probably not if we expect the happy scenarios will take less than 3 days?